sqlx/sqlx.package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@dataform/sqlx",
+  "description": "Dataform SQLX language tools.",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "dependencies": {
+    "moo": "^0.5.0"
```

Comment:
Is moo still a direct dependency of @dataform/core? If not, maybe remove it from core.package.json (otherwise these will inevitably get out of sync!). If it is, then maybe it shouldn't be (if you only need the types, you can still remove it).

Reply:
good call, done!